### PR TITLE
Allow reporting of places

### PR DESCRIPTION
**Pull request by mutaphysis**
- Extracted basic interface for reporting for future deduplication of code
- Added POST endpoint to report places
- Added simple UI for showing reports on admin page
- Allow dismissing reports or marking places as deleted (for now)

Report with:

`curl -X POST "https://www.wheelmap.pro/place-infos/report?id=b5f6Xx8CsbCKqsDWF&reason=invalid-place&message=This%20place%20SUCKS&appToken=<APP_TOKEN>"`

### CODE REVIEW

**Comment by mutaphysis** [comment]
Currently live on wheelmap.pro

**Comment by mutaphysis** [comment]
Screenshot 2019-07-12 at 18 00 49

Screenshot 2019-07-12 at 17 53 18

File: imports/server/collections/collections/PlaceInfos/registerMethods.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        throw new Meteor.Error(403, 'You have no rights to reject PlaceInfos');
+      }
+  
+      const place = PlaceInfos.findOne({ _id: placeId });
+      if (!place) {
+        throw new Meteor.Error(404, 'place not found');
+      }
+      
+      // approve, remove reports and unset moderationRequired
+      PlaceInfos.update(placeId, {
```
Comment (mutaphysis) [inline imports/server/collections/collections/PlaceInfos/registerMethods.ts]: -2: Refresh fastly cache here!

File: imports/server/collections/collections/PlaceInfos/registerMethods.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        throw new Meteor.Error(403, 'You have no rights to dismiss PlaceInfo Reports');
+      }
+  
+      const place = PlaceInfos.findOne({ _id: placeId });
+      if (!place) {
+        throw new Meteor.Error(404, 'place not found');
+      }
+  
+      // approve, remove reports and unset moderationRequired
+      PlaceInfos.update(placeId, {
```
Comment (mutaphysis) [inline imports/server/collections/collections/PlaceInfos/registerMethods.ts]: -2: Refresh fastly cache here!

**Comment by mutaphysis** [comment]
Regarding:
> Downgrade meteoreact:accounts to 1.2.3

@opyh did you forget to checkout the git submodule?

**Comment by opyh** [comment]
Argh, yep! Thanks for mentioning.

**Comment by mutaphysis** [comment]
No worries, I don't think it breaks anything at the moment.

**Comment by opyh** [comment]
Reverted the change on master, checked out and deployed :)

**Pull request merged**